Banking/src/main/java/com/example/Banking/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.example.Banking.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class ResourceNotFoundException extends Exception{
```

Comment:
This is nice! A custom exception. Before curly braces there should be a space "Exception {"

Reply:
Okay